jsk_perception/CMakeLists.txt

```diff
@@ -246,23 +246,25 @@ add_dependencies(hough_lines              ${PROJECT_NAME}_gencfg ${PROJECT_NAME}
 add_dependencies(rectangle_detector       ${PROJECT_NAME}_gencfg ${PROJECT_NAME}_gencpp)
 add_dependencies(color_histogram_sliding_matcher       ${PROJECT_NAME}_gencfg ${PROJECT_NAME}_gencpp)
 
+# FIXME: actually, roseus executable is usable only on Linux
+if(GNU)
```

Comment:
ok, 
```
diff --git a/roseus/roseus.cpp b/roseus/roseus.cpp
index 535d80f..046a54f 100644
--- a/roseus/roseus.cpp
+++ b/roseus/roseus.cpp
@@ -80,6 +80,16 @@
 #define string  eus_string
 
 #include "eus.h"
+
+extern "C" {
+byte *eus_get_string(register pointer s) {
+  if (isstring(s)) return(s->c.str.chars);
+  if (issymbol(s)) return(s->c.sym.pname->c.str.chars);
+  else error(E_NOSTRING);
+}
+}
+#define
```
solves the problem, the problem is I think `get_string` is defined in somewhere else. So next target is to find the location of `get_string`, other than https://github.com/euslisp/EusLisp/blob/master/lisp/c/makes.c#L67

If gdb works, I think we can find the location, if not, we'll check them by running `nm -A`, and we need to know which libraries are used in roseus, so I'd like to check by
```
ldd roseus/euslisp/roseus.so
``
but could not find ldd 
```
% brew install ldd       
Error: No available formula with the name "ldd" 
==> Searching for similarly named formulae...
This similarly named formula was found:
dcfldd ✔
To install it, run:
  brew install dcfldd ✔
==> Searching taps...
Error: No formulae found in taps.
```